pkg/types/utils.go

```diff
@@ -33,6 +33,14 @@ func SetDefaultLoggerConfig(cfgMode string, cfgFolder string, cfgLevel log.Level
 		if compress != nil {
 			_compress = *compress
 		}
+		
+		if LogFormat == "json" {
+			logFormatter = &log.JSONFormatter{TimestampFormat: time.RFC3339}
+		} else if LogFormat == "text" {
+			logFormatter = &log.TextFormatter{TimestampFormat: time.RFC3339, FullTimestamp: true, ForceColors: forceColors}
+		} else {
+			return fmt.Errorf("log format '%s' unknown", LogFormat)
```

Comment:
imo to not break backwards compat could we just default to `text` and issue a warning saying there was no value presented so default to `text`

```go
if LogFormat == "json" {
  logFormatter = &log.JSONFormatter{TimestampFormat: time.RFC3339}
} else {
   if LogFormat == "" {
       log.Warn("no log_format configured default to 'text' format")
   }
   logFormatter = &log.TextFormatter{TimestampFormat: time.RFC3339, FullTimestamp: true, ForceColors: forceColors}
}
```

Reply:
so if the user configures it will suppress the `warn`

Reply:
Also I think the above changes will fix the failing tests since it doesnt set to a value so you get a nil pointer

Reply:
Sounds reasonable, i will update the code accordingly. Thanks for the feedback!